locale/fr/about/index.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: about.hbs
+title: A propos
+trademark: Trademark
+---
+# A propos de Node.js&reg;
+
+En tant qu'environnement d'exécution JavaScript asynchrone et orienté événnement, Node est conçu
+pour générer des applications évolutives. Dans le "hello world" d'exemple
```

Comment:
Hard to traduct scalable that is an english word. I may have use scalable here to have a better understanding from readers. I think the idea was to explain that with nodejs we can build system that can adapt to performance need easily. It depends of the politics of the nodejs organization regarding to translating

Reply:
You're quite right. I think I even used scalable later on the page. I'll change this.